lib/ast/return.rb

```diff
@@ -20,4 +20,8 @@ def code(parameters = [], done_name = nil)
   def data
     @expression.data if @expression.respond_to? :data
   end
+
+  def validate(types)
+    @expression.validate(types) if @expression.is_a? Expression
+  end
```

Comment:
Return always seems to just pass on parameters without really doing much. I'm beginning to doubt it's necessary/helpful